custom/icds_reports/static/js/directives/awc-reports/awc-reports.directive.js

```diff
@@ -2649,7 +2649,7 @@ function AwcReportsController($scope, $http, $location, $routeParams, $log, DTOp
                 url: 'https://api.mapbox.com/styles/v1/dimagi/cj2rl1t0w001f2rnr0y8hfhho/tiles/{z}/{x}/{y}?access_token={apikey}',
```

Comment:
should this URL also be configurable? (it seems to be Dimagi specific)

Reply:
hmm. yeah probably, but i'd say let's address that if/when another org needs to host the dashboard.

Reply:
we should probably do that prior to handover with govt

Reply:
for sure, I imagine we'd bundle it with the key changes

Reply:
added an item to https://docs.google.com/document/d/19nEiTbJbPc7IPXmGVNAePyL6WiD0i8RmT6vM11CCBsI/edit